<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NDVEstimationDialog</class>
 <widget class="QDialog" name="NDVEstimationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0,0,0,0,0,0" columnminimumwidth="0,0,0,0,0,0,0,0,0,0,0">
     <item row="1" column="1" colspan="10">
      <widget class="QLabel" name="lblVariableName">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#0000ff;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="10">
      <widget class="QLabel" name="lblGridName">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#0000ff;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Variable:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Grid:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Search neighborhood:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QSpinBox" name="spinNCols">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="5" colspan="2">
      <widget class="QSpinBox" name="spinNRows">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="9">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>=</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="8">
      <widget class="QSpinBox" name="spinNSlices">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="10">
      <widget class="QLabel" name="lblSizeMetric">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#0000ff;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="7">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Kriging type:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QComboBox" name="cmbKType">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>SK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OK</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Mean for SK:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="6" colspan="5">
      <widget class="QCheckBox" name="chkUseSKMeanAsDefault">
       <property name="text">
        <string>use as default value (SK or OK)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="4">
      <widget class="QLineEdit" name="txtMeanForSK">
       <property name="text">
        <string>0.00</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Max. number of samples:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="spinNbSamples">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="lbl_56">
       <property name="text">
        <string>Variogram:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="9">
      <widget class="QFrame" name="frmVariogramPlaceholder">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>98</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDismiss">
       <property name="text">
        <string>Dismiss</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnDismiss</sender>
   <signal>clicked()</signal>
   <receiver>NDVEstimationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinNCols</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NDVEstimationDialog</receiver>
   <slot>updateMetricSizeLabels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinNRows</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NDVEstimationDialog</receiver>
   <slot>updateMetricSizeLabels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinNSlices</sender>
   <signal>valueChanged(int)</signal>
   <receiver>NDVEstimationDialog</receiver>
   <slot>updateMetricSizeLabels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRun</sender>
   <signal>clicked()</signal>
   <receiver>NDVEstimationDialog</receiver>
   <slot>run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateMetricSizeLabels()</slot>
  <slot>run()</slot>
 </slots>
</ui>
